DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -577,7 +577,7 @@ def connectToServer(self):
         offset = 0
 
         while True:
-            subset = self.DICOMwebClient.search_for_studies(offset=offset)
+            subset = self.DICOMwebClient.search_for_studies(offset=offset, fields=["StudyDescription"])
```

Comment:
Describe in a comment why this field is specified.

Reply:
GCP has `StudyDescription` as an [optional field](https://cloud.google.com/healthcare-api/docs/dicom#returned_results)

Reply:
Thank you. Please describe this in a comment in the source code.